src/app/reducers/area.reducer.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { AreaReducer } from './area.reducer';
+import { initialState as AreaInitialState, AreaState } from './../states/area.state';
+import * as AreaActions from './../actions/area.actions';
+import { AreaUI } from './../models/area.model';
+export type Action = AreaActions.All;
+
+describe('AreaReducer', () => {
+  it('AreaReducer: undefined action should return the default state', () => {
```

Comment:
Please reword this to `Undefined action should return the default state`.
The logs would automatically show `AreaReducer:  Undefined action should return the default state`